sql/mob_droplist.sql

```diff
@@ -17428,6 +17428,7 @@ INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15123,10);
 INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15140,10);
 INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15142,10);
 INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,16360,10);
+INSERT INTO `mob_droplist` VALUES (3223,0,0,1000,16017,212);
```

Comment:
From this page data: http://ffxidb.com/zones/193/bombast
It would seem that this drop would fall either into very common or common per the chart I linked you.

The data-sample we have is not that great, unfortunately, so I would say change it to common (150)

Reply:
Are we converting all droprates to the Eden / Limitbreak implementation of rarity bases? (I'm guessing yes since that is the retail accurate way)

Reply:
iirc Eden uses a non numeric entry in their drop tables: ie: Common, VC, Rare etc

And then based off those groupings affects the base droprates with the TH affects table that was shown in another MR recents.

Limitbreak changed all the droprates in a similar manner except instead of a string they used the: Common = 240, Rare = 100 etc method where everything that was determined to fit in those pools are assigned that pools respective numerical base droprate.

I think we talked about each of these in the ASB discord.

Reply:
Yea I believe either the Eden/Lb way is fine.

I think Edens is much easier. Just reference the rarity level and then you can adjust what those rarity levels mean elsewhere. Making it easier to modulize and customize if people want to.

Reply:
I stand in the oposite side. i would rather have all values be numeric. No need to just know the tier rates is one. the other for consistency, since the "paired" drops, like okote or dring dont have a % rate that fit a tier

As for the reason asking to change this to fit those tiers, its because the task of converting all drops to a fit proper tier is abysmal.
However, if adding or editing just a few, the effort to check them an do a good judjement on what tier they should have is much lower

Reply:
> I stand in the opposite side. i would rather have all values be numeric.

@xaver I was thinking maybe sql variables to sort of macro it. so the numbers don't go away, just a little abstracted. Like when you cast something to a lua variable and then reuse that variable in a dozen places. SQL can actually do that. I haven't mocked up an example yet been workin irl to much to _think_ about things.

Reply:
I think this is pretty much Edens way of doing it. But once again we can easily writeup a script to go throught he sql and fit the current data to a 'Rarity' as a first pass. Then as users PR things in that have drops, just request for them to verify actual drop rates. (Although Limitbreak did a vast amount of the work for us there up to ToAU) So we could start with their #s and then fit the rest.

Reply:
@The-Aerec I think we have a long way to go on manually correcting rates yet tho, even if a script replaced existing values to nearest official rate many would still be wrong. I'm sure we'll have many future PRs either way. We're kind of Cluttering crits PR now. If we need to we can make a discussion topic for further TH things if they aren't for the purpose of aiding his PR.